examples/pipelines/datacomp/components/filter_clip_score/fondant_component.yaml

```diff
@@ -1,6 +1,6 @@
 name: Filter CLIP score
 description: Component that filters out bad captions (Empty captions, Captions with weird characters, Captions that are dates)
-image: fndnt/filter_clip_score:dev
+image: fndnt/filter_clip_score:0.6.2
```

Comment:
We'll have to check if we still want to use the `fndnt` registry for the custom components in the examples.